service/conf.ml

```diff
@@ -229,10 +205,11 @@ let platforms ~profile ~include_macos opam_version =
       let[@warning "-8"] (latest :: previous :: _) =
         List.rev OV.Releases.recent
       in
-      let macos_distros = if include_macos then macos_distros else [] in
       let ovs = [ latest; previous ] in
       let releases = List.map make_release ovs in
-      releases @ macos_distros @ freebsd_distros
+      releases
+      |> List.append (if include_macos then macos_distros else [])
+      |> List.append (if include_freebsd then freebsd_distros else [])
```

Comment:
As an addition to the PR could you remove these two lines? To stop including Macos and FreeBSD distros on the 'minimal' set of platforms (used for local testing of `ocaml-ci-service`).